internal/archive/fsops.go

```diff
@@ -118,47 +118,64 @@ func walk(sourcePath string, baseDir string, archive *zip.Writer, ignore map[str
 			if info.IsDir() {
 				return filepath.SkipDir
 			}
-			return
+			return nil
 		}
 
-		header, err := zip.FileInfoHeader(info)
-		if err != nil {
-			return err
+		// Don't add the base folder to the zip
+		if info.IsDir() {
```

Comment:
I think you can simplify this to something like:
if info.IsDir() && filepath.Clean(path) == filepath.Clean(baseDir) {
	return nil
}

Reply:
Done